Test/logger/TickLogger.dfy

```diff
@@ -0,0 +1,42 @@
+// RUN: %dafny /verificationLogger:csv;LogFileName="%t-res.csv",Metric=ticks "%s"
+// RUN: %OutputCheck --file-to-check "%t-res.csv" "%s"
+
+// CHECK: TestResult\.DisplayName,TestResult\.Outcome,TestResult\.ResourceCount
+// CHECK-NEXT: Impl\$\$_module.__default\.ExampleWithSplits\$\$1,Passed,.*
```

Comment:
I'd prefer to bump the version of Boogie we depend on first so that we actually get the resource count metric in the `/xml` output, and then actually assert that we get a value here, rather than just asserting the header is there.

Reply:
Oh, yes, we definitely need to release Boogie and depend on that before merging this. And I agree about strengthening the check.

Reply:
There's now an additional test that fails. Once the new Boogie release is out, I'll incorporate it into this branch and that test should start passing.